website/content/en/docs/deployment/vanilla/guide.md

```diff
@@ -33,290 +25,44 @@ Option 2 targets customization and ability to pick and choose individual compone
+### (Optional) Delete Amazon EKS cluster
 
-Install the Katib official Kubeflow component:
+If you created a dedicated Amazon EKS cluster for Kubeflow using `eksctl`, you can delete it with the following command:
 
-```sh
-kustomize build upstream/apps/katib/upstream/installs/katib-with-kubeflow | kubectl apply -f -
+```bash
+eksctl delete cluster --region $CLUSTER_REGION --name $CLUSTER_NAME
 ```
 
-#### Central Dashboard
-
-Install the Central Dashboard official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/centraldashboard/upstream/overlays/kserve | kubectl apply -f -
-```
-
-#### Admission Webhook
-
-Install the Admission Webhook for PodDefaults:
-
-```sh
-kustomize build upstream/apps/admission-webhook/upstream/overlays/cert-manager | kubectl apply -f -
-```
-
-#### Notebooks
-
-Install the Notebook Controller official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/jupyter/notebook-controller/upstream/overlays/kubeflow | kubectl apply -f -
-```
-
-Install the Jupyter Web App official Kubeflow component:
-
-```sh
-kustomize build awsconfigs/apps/jupyter-web-app | kubectl apply -f -
-```
-
-#### Profiles and Kubeflow Access-Management (KFAM)
-
-Install the Profile controller and the Kubeflow Access-Management (KFAM) official Kubeflow
-components:
-
-```sh
-kustomize build upstream/apps/profiles/upstream/overlays/kubeflow | kubectl apply -f -
-```
-
-#### Volumes Web App
-
-Install the Volumes Web App official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/volumes-web-app/upstream/overlays/istio | kubectl apply -f -
-```
-
-#### Tensorboard
-
-Install the Tensorboards Web App official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/tensorboard/tensorboards-web-app/upstream/overlays/istio | kubectl apply -f -
-```
-
-Install the Tensorboard controller official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/tensorboard/tensorboard-controller/upstream/overlays/kubeflow | kubectl apply -f -
-```
-
-#### Training Operator
-
-Install the Training Operator official Kubeflow component:
-
-```sh
-kustomize build upstream/apps/training-operator/upstream/overlays/kubeflow | kubectl apply -f -
-```
-
-#### AWS Telemetry
-
-Install the AWS Kubeflow telemetry component. This is an optional component. See [Usage Tracking]({{< ref "/docs/about/usage-tracking.md" >}}) for more information
-
-```sh
-kustomize build awsconfigs/common/aws-telemetry | kubectl apply -f -
-```
-
-#### User namespace
-
-Finally, create a new namespace for the the default user. In this example, the namespace is called `kubeflow-user-example-com`.
-
-```sh
-kustomize build upstream/common/user-namespace/base | kubectl apply -f -
-```
-
-### Connect to your Kubeflow cluster
-
-After installation, it will take some time for all Pods to become ready. Make sure all Pods are ready before trying to connect, otherwise you might get unexpected errors. To check that all Kubeflow-related Pods are ready, use the following commands:
```

Comment:
We should keep this section before redirecting users toConnecting to your Kubeflow cluster

@ryansteakley